### PR TITLE
Rely on commander to argparse, handle new notebook

### PR DESCRIPTION
**Pull request by rgbkrk**
Switching us over to commander as well as bringing back the "launch new notebook" case.

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+import {
+  launchFilename,
+  launchNewNotebook,
+} from './launch';
 
 import { Menu } from 'electron';
 import { defaultMenu, loadFullMenu } from './menu';
 import { resolve } from 'path';
 
+const program = require('commander');
```
Comment (jdfreder) [inline src/main/index.js:12]: I :heart: commander

File: src/main/index.js [hunk 2]
```diff
@@ -13,17 +26,14 @@ app.on('window-all-closed', () => {
...
   // Get the default menu first
   Menu.setApplicationMenu(defaultMenu);
   // Let the kernels/languages come in after
   loadFullMenu().then(menu => Menu.setApplicationMenu(menu));
-  notebooks.filter(Boolean).forEach(f => launchFilename(resolve(f)));
+  if (notebooks <= 0) {
+    launchNewNotebook('python3');
+  } else {
+    notebooks.filter(Boolean).forEach(f => launchFilename(resolve(f)));
+  }
```
Comment (jdfreder) [inline src/main/index.js:38]: Much better

Comment (rgbkrk) [inline src/main/index.js:38]: I wondered about providing a `--kernel` argument, then realized it wouldn't make sense when you have a list of notebooks.

Comment (jdfreder) [inline src/main/index.js:38]: --kernel makes sense for the new notebook feature

**Pull request merged**